com.plexamp.Plexamp.metainfo.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id type="desktop">com.plexamp.Plexamp</id>
+    <launchable type="desktop-id">com.plexamp.Plexamp.desktop</launchable>
+    <name>Plexamp</name>
+    <summary>Plexamp - A beautiful Plex music player for audiophiles, curators, and hipsters.</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-Proprietary=https://www.plex.tv/about/privacy-legal/plex-terms-of-service/</project_license>
+    <developer_name>Plex</developer_name>
+    <url type="homepage">https://plexamp.com</url>
+    <description>
+        <p>Plexamp is a small, highly opinionated music player for Plex Media Server.</p>
+        <p>A Plex Pass is required to use Plexamp.</p>
```

Comment:
As noted above, please put the "You'll need the Plex Media Server and a Plex Pass to use this app" info here and also something about this not being the upstream supported version, the form of words we usually see is something like "NOTE: This wrapper is not verified by, affiliated with, or supported by Plex."